lib/local-server.js

```diff
@@ -131,7 +131,8 @@ module.exports = function (gulp, gutil) {
             pluginResPath,
             pluginWebPath,
             angularLiSrcPath,
-            opts.localServerDir
+            opts.localServerDir,
+            '.'
```

Comment:
A better solution would be serve the activecast folder as "activecast" but since we use gulp-connect which wraps "connect" we cannot do this.